### ISSUE TITLE
[REQ] Support gzip compression

### ISSUE THREAD

**Issue by slash-arun** [OpenAPITools/openapi-generator#4165]
### Is your feature request related to a problem? Please describe.

useGzipFeature is supported in Java, need the same for python

## Describe the solution you'd like

Both content-encoding and accept-encoding should support gzip

**Comment by wing328** [comment]
Right. Only Java supports that feature at the moment. We welcome implementation to extend the gzip support to other languages.
